### DISCUSSION TITLE
Is there an alternative fastapi version for M1 architecture?

### DISCUSSION THREAD

**Discussion by ShimronAlakkal** [fastapi/fastapi#4484] [Questions]
Shows an error while running `unicorn main:app --reload` command.

**error shown**
```bash
mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64e')), '/usr/lib/__init__.cpython-310-darwin.so' (no such file)
```

**Comment by acidjunk** [comment]
Maybe this will help: https://www.renedohmen.nl/running-python-on-mac-m1-natively/

I'm running it natively for a couple of months now.

**Reply by ShimronAlakkal** [reply]
Thanks. I'll try it out.
